### ISSUE TITLE
Z-Fighting in Furnace, Alloy Furnace, Melter Models When on Top of Non-Solid Blocks

### ISSUE THREAD

**Issue by charlie-e09ae35d** [MinestrapTeam/Minestrappolation-4#72]
Minestrappolation-1.8-4.0.0.nightly_233
![dbaaadf23f656416ce4c51d29e505072](https://cloud.githubusercontent.com/assets/13768838/9255415/732db376-41e9-11e5-9800-7d65834f5103.png)
[video demo](http://gfycat.com/MenacingElectricAchillestang)
z-fighting visible on the bottom of the blocks front face,
affected blocks are furnace, alloy furnace, melter,

only happens when machine is on top of blocks like: hopper, glass, beacon, crafting table, trap doors, leaves, glass pane, carpenters bench, alloy furnace, melter, crusher,

changing zero values to small positive ones in those files fixed it for me, but i dont know if it is a proper fix

```
assets/minecraft/models/block/furnace.json
12
-     "to": [ 13, 0, 1 ],
+     "to": [ 13, 0.001, 1 ],
assets/minecraft/models/block/lit_furnace.json
12
-     "to": [ 13, 0, 1 ],
+     "to": [ 13, 0.001, 1 ],
assets/ministrapp/models/block/alloy_furnace.json
134
-     "to": [ 13, 0, 1 ],
+     "to": [ 13, 0.001, 1 ],
assets/ministrapp/models/block/alloy_furnace_active.json
134
-     "to": [ 13, 0, 1 ],
+     "to": [ 13, 0.001, 1 ],
assets/ministrapp/models/block/melter.json
265
-     "to": [ 13, 0, 1 ],
+     "to": [ 13, 0.001, 1 ],
assets/ministrapp/models/block/melter_active.json
265
-     "to": [ 13, 0, 1 ],
+     "to": [ 13, 0.001, 1 ],
```

**Comment by Delocuro** [comment]
Yep, that's a known issue that I never got around to fixing, but I can try to do that for the next update. Basically what's happening is that if the block below the Furnace, Alloy Furnace, etc is not solid, then the top face still tries to render, which intersects with the thin bottom texture. Increasing the values to 0.001 does indeed solve the problem, so I'll do that for 4.1.1.

**Comment by Delocuro** [comment]
Merged issue into #87.
